Code/Framework/AzFramework/AzFramework/Terrain/TerrainDataRequestBus.h

```diff
@@ -161,6 +161,10 @@ namespace AzFramework
                 SurfacePointListFillCallback perPositionCallback,
                 Sampler sampleFilter = Sampler::DEFAULT) const = 0;
 
+            //! Returns the number of samples for a given region and step size.
```

Comment:
nit: Since the pair isn't clearly an x/y count, at least the comment should mention that.

Reply:
Updated comment